### ISSUE TITLE
set homekit transition control flooding HB logs

### ISSUE THREAD

**Issue by jordanearle** [ebaauw/homebridge-hue#981]
#### Issue
Hi there, I have had Hue on Homebridge running smoothly for a while now but I've noticed my logs are being flooded with the same message (seemingly per bulb) and I'm not sure how to disable the message? It's not causing any issues, but makes the log hard to parse.

#### Log Messages
```
[07/05/2021, 16:26:01] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 13: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 8: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 13: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 8: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 13: set homekit transition control to ""
```

#### Debug Files
[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/6442724/homebridge-hue.json.gz)

**Comment by jordanearle** [comment]
<!--

#### Issue
Hi there, I have had Hue on Homebridge running smoothly for a while now but I've noticed my logs are being flooded with the same message (seemingly per bulb) and I'm not sure how to disable the message? It's not causing any issues, but makes the log hard to parse.

#### Log Messages
```
[07/05/2021, 16:26:01] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 13: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:01] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 8: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 13: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 9: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 1: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 11: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 8: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 12: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 7: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 10: set homekit transition control to ""
[07/05/2021, 16:26:06] [Hue] Color temperature light 13: set homekit transition control to ""
```

#### Debug Files
[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/6442724/homebridge-hue.json.gz)

**Comment by ebaauw** [comment]
The message is issued when HomeKit (or the Home app?) requests the adaptive lighting state from the _Lightbulb_ service.  Normally it does so only once; I don't know why you're getting so many requests.  What HomeKit apps are you using?  Are you controlling the lights from the Homebridge UI or using some integration?

**Comment by jordanearle** [comment]
> The message is issued when HomeKit (or the Home app?) requests the adaptive lighting state from the _Lightbulb_ service.  Normally it does so only once; I don't know why you're getting so many requests.  What HomeKit apps are you using?  Are you controlling the lights from the Homebridge UI or using some integration?

Hey, I'm not using any other HomeKit apps aside from the Home app. 

Hue is the only lighting integration I have on Homebridge. I control them using the Home app. 

The only other relevant thing is that I'm using a bunch of IKEA Tradfri bulbs? They're all linked to the Hue hub directly which is then linked to the Hue plugin. 

Is there a config I'm missing for the IKEA bulbs?

Or any other ideas?

**Comment by willipreuk** [comment]
I have the same issue, also using Tradfri bulbs.

**Comment by ebaauw** [comment]
Afaik, the message, or rather the behaviour of the Home app, does not depend on the type of light.  As I said above, these messages are expected when you start Home, but shouldn't be "flooding" the log file.

If these message really are appearing all the time, please capture and attach a debug log file with the `DEBUG` environment variable set to `*`, so HAP-NodeJS logs the incoming requests from Home, and of course with the Homebridge `DEBUG` setting, so Homebridge Hue does extensive logging.  See the _Homebridge Settings_  under the Homebridge UI.
